packages/forms/src/widgets/MultiSelectTagWidget/MultiSelectTagWidget.js

```diff
@@ -155,6 +157,16 @@ class MultiSelectTagWidget extends React.Component {
 		this.updateSuggestions(value);
 	}
 
+	onCaretClick() {
+		const input = this.component.querySelector('input');
+		if (this.state.isFocused) {
+			input.blur();
+		} else {
+			input.focus();
+		}
+		this.setState({ isFocused: !this.state.isFocused });
```

Comment:
When you wish to change the sate based on its previous value you have to use the

Reply:
You have to use an updater function, check react documentation about why.